server/user_account/views.py

```diff
@@ -145,18 +142,29 @@ def remove_token(self, request): # pylint: disable=no-self-use
                         status=status.HTTP_200_OK)
 
 
-class AccessAllClients(generics.GenericAPIView):
-    queryset = CustomUser.objects.values(
-        'user_name',
-        'first_name',
-        'last_name',
-        'role',
-        'is_active',
-        'email',
-        )
-    http_method_names = ['get']
-
+class AccessAllClients(viewsets.ModelViewSet):
+    @api_view(['Get'])
+    @permission_classes([IsAuthenticated])
     def get(self, request):
```

Comment:
Also I'm not even sure how this function is working because you are declaring your class based view as a modelViewSet and inside of it you are creating a function based view which does not make sense. I know the concept about class based views and function based views are confusing, If you want we can get on a call and I"ll be glad to help you clariy some stuff.